lib/features/onboarding/ui/pages/otp_page.dart

```diff
@@ -53,9 +55,10 @@ class _OtpPageState extends State<OtpPage> {
             child: Column(
               crossAxisAlignment: CrossAxisAlignment.start,
               children: [
-                const OnboardingTitle(
+                OnboardingTitle(
                   title: OnboardingConstants.otpHeading,
-                  subTitle: OnboardingConstants.otpSentText,
+                  subTitle:
+                      ' ${OnboardingConstants.otpSentText}${widget.number}.',
```

Comment:
Why there is a Space before Constant-Text, and also need to optimise this for better readability as 
```
final subTitle = '${OnboardingConstants.otpSentText}${widget.number}.'
```
```suggestion
                     subTitle,
```

Reply:
This is the small mistake so i have removed Space before Constant-Text.